sklearn/metrics/pairwise.py

```diff
@@ -1938,7 +1938,7 @@ def _dist_wrapper(dist_func, dist_matrix, slice_, *args, **kwargs):
 
 def _parallel_pairwise(X, Y, func, n_jobs, **kwds):
     """Break the pairwise matrix in n_jobs even slices
-    and compute them in parallel."""
+    and compute them using threading."""
```

Comment:
Nit: I think [multithreading](https://en.wikipedia.org/wiki/Multithreading_(computer_architecture)) is the common term:

```suggestion
    and compute them using multithreading."""
```